coveo/ui-kit
# feat(quantic): fix-facets-selected-state

### Related Issue
[SFINT-3963](https://coveord.atlassian.net/browse/SFINT-3963)

### Changes Proposed
Added a isChecked @api value to each types of facet values that the parent can set according to the current state of the facet from headless.

~Also allowed clicking on the whole row of a facet value instead of just the checkbox in the facet.~ Removed as out-of-scope

### How to test
Select one or multiple facet values and then de-select them using the breadcrumb component, the checkbox next to the right facet value should be un-selected.
